testing/bdd/features/styles/ebuttd_fontsize_conversion.feature

```diff
@@ -25,7 +25,7 @@ Feature: Compute fontSize on a single EBU-TT Live element
     Examples:
     | xml_file                      | extent | cell_resolution | S1_value | S2_value | S3_value | S4_value  | S5_value | S6_value |
     | ebuttd_fontsize_convert.xml   |        | 32 15           | 100%     | 100%     | 100%     | 100%      | 100%     | 100%     |
-    | ebuttd_fontsize_convert.xml   |        | 32 15           | 50%      | 100%     | 100%     | 100%      | 1c       | 5c       |
+    | ebuttd_fontsize_convert.xml   |        | 32 15           | 50%      | 100%     | 100%     | 100%      | 1c       | 1c       |
```

Comment:
I don't understand why this change helps? Since the test is simply tht the converted document is valid, any number of `c` could go in the S6_value, right?